### PR TITLE
New EA Eventbrite Origin work

### PR DESCRIPTION
**Pull request by sc0ttkclark**
https://central.tri.be/issues/96078 and the TEC plugin issues linked within (client-side)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [New.php:184](https://github.com/moderntribe/the-events-calendar/blob/9ad23d1be6552f68ad689fdb8af192f798618a69/src/Tribe/Aggregator/Tabs/New.php#L184)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [status.php:208](https://github.com/moderntribe/the-events-calendar/blob/9ad23d1be6552f68ad689fdb8af192f798618a69/src/admin-views/aggregator/status.php#L208)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$notes' | [status.php:212](https://github.com/moderntribe/the-events-calendar/blob/9ad23d1be6552f68ad689fdb8af192f798618a69/src/admin-views/aggregator/status.php#L212)

> via jenkins/codesniffer-phpcs

**Comment by bordoni** [comment]
@sc0ttkclark if you can merge the latest code from `bucket/tickets-revival` into this, so we can have our Tests (Travis-CI) passing.

**Comment by sc0ttkclark** [comment]
Done

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [New.php:184](https://github.com/moderntribe/the-events-calendar/blob/9144b0b89cd110a6e21814b3e8e3c55f010d02fe/src/Tribe/Aggregator/Tabs/New.php#L184)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [status.php:208](https://github.com/moderntribe/the-events-calendar/blob/9144b0b89cd110a6e21814b3e8e3c55f010d02fe/src/admin-views/aggregator/status.php#L208)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$notes' | [status.php:212](https://github.com/moderntribe/the-events-calendar/blob/9144b0b89cd110a6e21814b3e8e3c55f010d02fe/src/admin-views/aggregator/status.php#L212)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Settings.php [hunk 1]
```diff
@@ -126,6 +127,98 @@ public function is_fb_credentials_valid( $time = null ) {
...
+			add_query_arg(
+				'action',
+				'disconnect-eventbrite',
+				$url
+			),
+			'disconnect-eventbrite'
+		);
+	}
+
+	public function is_eb_credentials_valid( $time = null ) {
```
Comment (borkweb) [inline src/Tribe/Aggregator/Settings.php]: Can you add a docblock to this and reference/describe the `$time` param?

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Comment refers to a TODO task | [New.php:184](https://github.com/moderntribe/the-events-calendar/blob/a52917e7c938e4725593b9a76a5d37cc519c1302/src/Tribe/Aggregator/Tabs/New.php#L184)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [status.php:208](https://github.com/moderntribe/the-events-calendar/blob/a52917e7c938e4725593b9a76a5d37cc519c1302/src/admin-views/aggregator/status.php#L208)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$notes' | [status.php:212](https://github.com/moderntribe/the-events-calendar/blob/a52917e7c938e4725593b9a76a5d37cc519c1302/src/admin-views/aggregator/status.php#L212)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Settings.php (hunk 1 above)
Comment (sc0ttkclark) [reply src/Tribe/Aggregator/Settings.php]: ✅

**Pull request merged**